ukis_pysat/data.py

```diff
@@ -87,7 +87,7 @@ def __init__(self, datahub, catalog=None, url=None):
             self.api = sentinelsat.SentinelAPI(
                 self.user,
                 self.pw,
-                "https://scihub.copernicus.eu/apihub",
+                "https://apihub.copernicus.eu/apihub",

```

Comment:
i found the error as well and changed the OA url as you suggested.

Reply:
Good!